lib/person.rb

```diff
@@ -0,0 +1,9 @@
+class Person
+  attr_accessor :name,:list_of_borrowed_book
+
+  def initialize(options ={})
+    @name = options[:name]
+    @list_of_borrowed_book = []
```

Comment:
How and when is this list updated? I can't see a method for actually adding books to this list.